docs/metrics.md

```diff
@@ -69,8 +69,9 @@ To enable default metrics set you need to specify the `metrics_enable` flag or j
 CLI
 
    ```bash
-         docker run --rm -d -p 9000:9000 -p 8000:8000 openvino/model_server:latest \
-                --model_name resnet --model_path gs://ovms-public-eu/resnet50  --port 9000 \
+         wget -N https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.{xml,bin} -P models/resnet50/1
```

Comment:
maybe download it once to workspace/models/resnet50/1 and use in every docker command?